test/parseUtils.js

```diff
@@ -26,8 +31,29 @@ describe('parseBundle', function () {
 
   it("should parse invalid bundle and return it's content and empty modules hash", function () {
     const bundleFile = `${BUNDLES_DIR}/invalidBundle.js`;
-    const bundle = parseBundle(bundleFile);
+    const bundle = parseBundle(bundleFile, {});
     expect(bundle.src).to.equal(fs.readFileSync(bundleFile, 'utf8'));
     expect(bundle.modules).to.deep.equal({});
   });
+
+  Object.keys(COMPRESSIONS)
+    .forEach(compressionType => {
+      it(`should parse compressed ${compressionType} bundle`, function () {
+        const {extension, algorithm} = COMPRESSIONS[compressionType];
+        const bundleFile = `${BUNDLES_DIR}/validBundleWithArrowFunction.js`;
+        const compressedBundleFile = `${bundleFile}.${extension}`;
+        const expectedModules = JSON.parse(fs.readFileSync(`${BUNDLES_DIR}/validBundleWithArrowFunction.modules.json`));
+        if (!zlib[algorithm]) {
```

Comment:
We don't need this condition - it swallows valid error and we won't notice it.

Reply:
For brotli, node version needs to be >= v11.7.0. Thus I put it to pass ci-checks which run on node 8. How do you suggest handling it?

Reply:
> For brotli, node version needs to be >= v11.7.0
We need to mention about it in the docs and properly handle in the code.

Reply:
> How do you suggest handling it?

Reply:
> We need to mention it in the docs and properly handle in the code.

Brotli compression works for version >= v11.7.0,  so it is kind of an already set expectation for brotli assets. Would you prefer it including in docs anyway?

Reply:
Yes. Maybe compression is done with some external tool instead of Node.js.